src/main/java/de/tum/in/www1/bamboo/server/parser/domain/Issue.java

```diff
@@ -1,50 +1,84 @@
 package de.tum.in.www1.bamboo.server.parser.domain;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class Issue {
 
-    private String classname;
+    // Path of to the source file using unix file separators
+    private String file;
 
-    private String type;
+    private Integer startLine;
 
-    private String priority;
+    private Integer endLine;
+
+    private Integer startColumn;
+
+    private Integer endColumn;

```

Comment:
Which of the values like `endLine`, `startLine`, etc. are mandatory? Or are all of them optional?

Reply:
Spotbugs: startLine
CheckStyle: startLine, startColumn
PMD: startLine, endLine, startColumn, endColumn
I'll add this as a comment